### DISCUSSION TITLE
Updating jsonb fields

### DISCUSSION THREAD

**Discussion by renom** [aarondl/sqlboiler#1371] [Q&A]
Is it possible to perform a jsonb update query on a model?
```
UPDATE test SET data = jsonb_set(data, '{name}', '"my-other-name"');
```

**Comment by stephenafamo** [comment]
SQLBoiler does not have convenient query mods for JSONB operations at the moment.

**Reply by renom** [reply]
So the only way is to use raw sql queries?
